modules/development/pages/kubernetes-vm.adoc

```diff
@@ -0,0 +1,68 @@
+= Kubernetes VM
+:toc:
+:toclevels: 3
+
+WARNING: This page is a work in progress.
+
+. Copy the kubernetes VM image from `node-images` into the management VM.
++
+[source,bash]
+----
+scp node-images/output-kubernetes-vm-qemu/kubernetes-vm-x86_64.qcow2 root@192.168.0.2:~/
+----
+. Login to the Management VM
+
+There are two options to get started. If you're still on the 1st Hypervisor, you can login using the isolated network. The address of the management-vm will be 192.168.0.2 or whatever value was assigned in the management-vm configuration steps.
```

Comment:
```suggestion
There are two options to get started. If you're still on the first Hypervisor, you can login using the isolated network. The management-vm will resolve from `/etc/hosts`, its IP may also be found by running `virsh net-dhcp-leases isolated`.
```

Reply:
I think if there isn't another option presented here, then it would be better to remove the phrasing "If you're still on the first hypervisor." What if I'm not on the first hypervisor, then what do I do here?